hummingbot/connector/parrot.py

```diff
@@ -55,44 +60,65 @@ async def get_campaign_summary(exchange: str, trading_pairs: List[str] = []) ->
     return results
 
 
-async def get_market_snapshots(market_id: int):
+async def get_market_snapshots(market_id: int, timestamp: int = 1657747860000):
     async with aiohttp.ClientSession() as client:
-        url = f"{PARROT_MINER_BASE_URL}market_snapshots/{market_id}?aggregate=1m"
+        url = f"{PARROT_MINER_BASE_URL}user/single_snapshot?client_id=00000000000000000000000000000000000000000&market_id={market_id}&timestamp={timestamp}&aggregate_period=60"
         resp = await client.get(url)
         resp_json = await resp.json()
     return resp_json
 
 
-async def get_active_campaigns(exchange: str, trading_pairs: List[str] = []) -> Dict[int, CampaignSummary]:
+async def get_active_campaigns(exchange: str, trading_pairs=None) -> Dict[int, CampaignSummary]:
```

Comment:
this change is the same, if you don't pass any value it will return an empty list, we should restore it

Reply:
This is possibly because PyCharm recommends to use the construct implemented rather than the 'None' in argument